docker/Dockerfile

```diff
@@ -42,19 +39,23 @@ RUN rm /tmp/search-guard-6-6.8.6-25.5.zip
 # Add default roles
 ADD sg_roles_mapping.yml /usr/share/elasticsearch/plugins/search-guard-6/sgconfig/sg_roles_mapping.yml
 
+# Allow read-only anonymous access
+RUN sed -i "s/anonymous_auth_enabled: false/anonymous_auth_enabled: true/" /elasticsearch/plugins/search-guard-6/sgconfig/sg_config.yml
+
 # Change log level INFO->DEBUG
 
 #RUN TMP_FILE=`mktemp /tmp/config.XXXXXXXXXX` && \
 #    sed -e "s/INFO/INFO/" /elasticsearch/config/logging.yml > $TMP_FILE && \
 #    mv $TMP_FILE /elasticsearch/config/logging.yml
 
-# Define working directory.
-WORKDIR /data
+# Define mountable directories.
+VOLUME ["/elasticsearch/data"]
 
+# Define working directory.
+WORKDIR /elasticsearch/data
 
 RUN adduser -D -u 1000 esuser
 RUN chown -R esuser /elasticsearch
-USER esuser
```

Comment:
I don't like this solution. I mean it is better to use the `RUN` docker file instruction than `su-exec`. https://docs.docker.com/engine/reference/builder/#user

So I would not modify the docker-entrypoint.sh at all.